src/DatePicker/DatePicker.js

```diff
@@ -180,14 +180,14 @@ const DatePicker = React.createClass({
 
   getInitialState() {
     return {
-      date: this._isControlled() ? this._getControlledDate() : this.props.defaultDate,
+      date: this.isControlled() ? this._getControlledDate() : this.props.defaultDate,
```

Comment:
@mbrookes I think this `isControlled()` migration is only partially done. I'm getting error on docs site `Uncaught TypeError: this.isControlled is not a function` and sill see `_isControlled()`